operators/churner/cmd/Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM golang:1.21.1-alpine3.18 as builder
 RUN apk add --no-cache make musl-dev linux-headers gcc git jq bash
 
 # build node with local monorepo go modules
-COPY ./operators/churner /app/churner
+COPY ./operators/churner /app/operators/churner
```

Comment:
How about `COPY ./operators /app/operators`, which will work for upcoming ejector